### PR TITLE
feat(core): Email recipients on resource shared

### PR DESCRIPTION
**Pull request by ivov**
https://linear.app/n8n/issue/PAY-1149

### CODE REVIEW

File: packages/cli/src/config/schema.ts [hunk 1]
```diff
@@ -846,6 +846,18 @@ export const schema = {
...
+					format: String,
+					default: '',
+					env: 'N8N_UM_EMAIL_TEMPLATES_WORKFLOW_SHARED',
+				},
+				credentialsShared: {
+					doc: 'Overrides default HTML template for notifying that credentials were shared (use full path)',
+					format: String,
+					default: '',
+					env: 'N8N_UM_EMAIL_TEMPLATES_CREDENTIALS_SHARED',
+				},
```
Comment (netroy) [inline packages/cli/src/config/schema.ts:860]: I've always wondered why we do it this way. can we not ask users to overwrite the files by using docker volumes, or even extending the official images with custom templates.
I believe that the burden of customization related code should lie on the party making those customizations.

**Review by netroy — APPROVED** [review]
LGTM

File: packages/cli/src/config/schema.ts (hunk 1 above)
Comment (ivov) [reply packages/cli/src/config/schema.ts:860]: @maspio Do you happen to know if embed customers usually need to customize email templates?

**Comment by janober** [comment]
Got released with [`n8n@1.26.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.26.0)

**Pull request merged**